src/Model/GServer.php

```diff
@@ -2048,19 +2051,22 @@ public static function setProtocol(int $gsid, int $protocol)
 	 *
 	 * @param int $gsid Server id
 	 * @return int
-	 * @throws Exception
+	 * @throws InvalidArgumentException If $gsid is not valid
+	 * @throws ResultException If $gsid cannot be found in table
 	 */
 	public static function getProtocol(int $gsid): int
 	{
-		if (empty($gsid)) {
-			return null;
+		if ($gsid < 1) {
```

Comment:
Why are you checking for `<1`? The `empty` check is completely okay.

Reply:
See above, `gserver.id` can NEVER be below 1.